intel-retail/automated-self-checkout
# fix: Add code to download yolov5 FP32-INT8

Closes #296

## PR Checklist



## What are you changing?


## Issue this PR will close

close: #296

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable
1. remove the old model folder `sudo rm -rf configs/opencv-ovms/models/2022/yolov5s`
2. download the new models `download_models/downloadOVMSModels.sh`
3. make sure the model is downloaded into both FP32_INT8 and FP16_INT8 folders.

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
